### PR TITLE
411 offer letter

### PR DESCRIPTION
**Pull request by mohitgusain**
- [x] Offer letter field added to the Send for approval model

- [x] Status changed from pending to Send for approval

### CODE REVIEW

File: config/constants.php [hunk 1]
```diff
@@ -109,7 +109,7 @@
             ],
             'sent-for-approval' => [
                 'label' => 'sent-for-approval',
-                'title' => 'Pending',
+                'title' => 'Sent-For-Approval',
```
Comment (rathorevaibhav) [inline config/constants.php]: Title should be `Sent for approval`

File: resources/views/hr/application/index.blade.php [hunk 2]
```diff
@@ -79,6 +71,15 @@
                 @endif
                 </a>
             </li>
+            <li class="nav-item">
```
Comment (rathorevaibhav) [inline resources/views/hr/application/index.blade.php]: what does this change mean?

File: config/constants.php (hunk 1 above)
Comment (mohitgusain) [reply config/constants.php]: Ok.

File: resources/views/hr/application/index.blade.php (hunk 2 above)
Comment (mohitgusain) [reply resources/views/hr/application/index.blade.php]: Nothing else, but I thought just status should be in also in serial after one by one. Should I change it or make it as earlier?

**Pull request closed without merging**